spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -256,7 +254,7 @@ def _measure_length(self, im_data, p_lst, idx):
 
     def _measure_diameter(self, im_data, p_lst, idx):
         area_lst = [np.sum(im_data[:, :, zz]) * np.cos(self.angles[zz]) * p_lst[0] * p_lst[1] for zz in range(im_data.shape[2])]
-        diameter_cur = 2 * np.sqrt(max(area_lst) / (4 * np.pi))
+        diameter_cur = 2 * np.sqrt(max(area_lst) / np.pi)
```

Comment:
This change also makes sense to me, as it lines up with what I wrote in my test:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/6f388046fa7d21d4315513eba8a2ec0b94267adf/testing/cli/test_cli_sct_analyze_lesion.py#L40-L46

My best guess is that the original `4 * pi` equation was derived from the formula for the area of a sphere?

```
   a = 4*pi*r^2 
-> a = 4*pi*(d/2)^2 
-> d = 2*sqrt(a/4*pi) 
```

But, the `max(area_lst)` used in this formula is computed using the 2D area, _not_ the 3D area:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/6f388046fa7d21d4315513eba8a2ec0b94267adf/spinalcordtoolbox/scripts/sct_analyze_lesion.py#L256

So, I agree that your change makes sense.